package.json

```diff
@@ -75,17 +88,22 @@
     "@govtechsg/oa-verify": "^7.4.3",
     "@govtechsg/open-attestation": "^5.3.4",
     "@govtechsg/token-registry": "^2.5.2",
+    "@walrus/rollup-plugin-shebang": "^1.0.1",
     "ajv": "^8.4.0",
     "ajv-formats": "^2.1.0",
+    "builtin-modules": "^3.2.0",
     "chalk": "^4.1.1",
     "debug": "^4.3.1",
     "ethers": "^5.4.1",
     "ethers-aws-kms-signer": "^1.3.2",
     "inquirer": "^8.0.0",
     "mkdirp": "^1.0.4",
     "node-fetch": "^2.6.1",
+    "rdf-canonize": "^3.0.0",
```

Comment:
use https://github.com/rollup/plugins/tree/master/packages/commonjs#ignoretrycatch instead of adding unecessary dependencies

Reply:
can elaborate?

I did as you recommended and the warning for rds-cannoize `(!) Unresolved dependencies
https://rollupjs.org/guide/en/#warning-treating-module-as-external-dependency` disappears

Reply:
How did you know that the related dependency is inside a try catch block?

Reply:
ok  i have removed the unused deps

Reply:
I know because I've seen some of those dep  when I debugged long ago